src/test/java/com/basho/riak/client/core/operations/itest/ITestBase.java

```diff
@@ -313,4 +318,22 @@ public static boolean assureIndexExists(String indexName) throws InterruptedExce
     public static Namespace defaultNamespace() {
         return new Namespace( testBucketType ? bucketType : BinaryValue.createFromUtf8(Namespace.DEFAULT_BUCKET_TYPE), bucketName);
     }
+
+    protected static void assertFutureSuccess(RiakFuture<?, ?> resultFuture)
```

Comment:
I guess that's true, but it leads to slightly different behavior though - if an assertTrue() fails, it will just say "assertion failed"  Failing an assertEquals throws a comparison failure with large nice message.